setup.py

```diff
@@ -49,7 +49,7 @@ def get_version():
             'autopep8',
             'mock; python_version < "3.3"',
             'mypy; python_version >= "3.3"',
-            'pep8',
+            'pycodestyle == 2.2.0',
```

Comment:
These are `extras_requires` for `dev` (aka development), not `install_requires`. Consumers won't need to adhere to these versions. This prevents spontaneous build failures though.

In a future PR I'm going to add exact versions for all of our `extras_requires`.